docs/CONTRIBUTING.md

```diff
@@ -229,7 +229,40 @@ Add "WIP" (Work in Progress) or "Draft" to the title if the PR is not yet ready
    git branch -d <branch-name>
    ```
 
-## Developer Certification of Origin (DCO)
+## Choosing the correct branch when contributing
+
+New writing goes to the `main` branch and can be viewed on the Releases -> development dropdown menu.
+We have staging and production levels for our documentation which are as follows:
+
+1. Development documentation aka staging (build of `main` branch) - [link](https://main.lifecycle.keptn.sh)
+2. Official documentation aka production (build of `page` branch) - [link](https://lifecycle.keptn.sh)
+
+Let's take a look what happens when your changes are merged in `main` and `page` branch respectively.
+
+### Development documentation (Staging)
+
+This page reflects the current development status of the documentation.
+It will be built regularly and can be easily accessed.
+
+* build: on each push to `main` with documentation changes
+* build-environment: main
+* config folder: [main](./config/staging/)
+
+It should allow bleeding-edge users and contributors to see the current state and help with debugging etc.
+
+### Official documentation (Production)
+
+This documentation set contains all released versions of KLT and is stored in an orphaned branch called `page`.
```

Comment:
```suggestion
This documentation set contains documentation for the current KLT release
and is stored in an orphaned branch called `page`.
When a new KLT version is released,
the content of the `development` branch is rolled to this branch.
Note also that doc modifications made through the
"Edit this page" facility are merged directly into this branch.

This uses the `latest` label so that links to a doc page
remain valid across software and documentation updates. 
```